.mailmap

```diff
@@ -99,3 +99,9 @@ Yash Sherry <yash14123@iiitd.ac.in>
 endolith <endolith@gmail.com>
 ArjitJ <32598699+ArjitJ@users.noreply.github.com>
 Kevin Sitek <sitek.kevin@gmail.com>
+Ross Lawrence <lawreros@gmail.com> Lawreros <lawreros@gmail.com>
+Ross Lawrence <lawreros@gmail.com>  Ross Lawrence <52179159+Lawreros@users.noreply.github.com>
```

Comment:
Isn't this repeated?

Reply:
No, it is not repeated, `Lawreos` to ` Ross Lawrence` and then link the different email